app/grandchallenge/reader_studies/models.py

```diff
@@ -879,6 +977,27 @@ def statistics(self):
             "questions": questions,
         }
 
+    @cached_property
+    def values_for_interfaces(self):
+        cache_key = f"{self.slug}-{self.modified.timestamp()}"
```

Comment:
There probably won't be a clash but ideally the key should include the app and model name